elasticsearch_dsl/_async/document.py

```diff
@@ -113,7 +143,13 @@ async def get(cls, id, using=None, index=None, **kwargs):
         return cls.from_es(doc)
 
     @classmethod
-    async def exists(cls, id, using=None, index=None, **kwargs):
+    async def exists(
+        cls,
+        id: str,
+        using: Optional[AsyncUsingType] = None,
+        index: Optional[str] = None,
+        **kwargs: Any,
```

Comment:
FYI I am not worrying about kwargs at this time, leaving most of them as `Any` to be reviewed later.